### PR TITLE
f-cookie-banner@v4.6.2 - update static cookie banner readme example

### PR DESCRIPTION
**Pull request by jamieomaguire**
v4.6.2
------------------------------
*October 11, 2022*

### Updated
- `README.md` to include `@justeat/fozzie` CSS links in static cookie banner example and explain why it is required to work

### CODE REVIEW

**Review by ashleynolan — DISMISSED** [review]

File: packages/components/organisms/f-cookie-banner/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v4.6.2
+------------------------------
+*October 11, 2022*
+
+### Updated
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-cookie-banner/CHANGELOG.md]: Minor, should be `changed`

**Review by juan-amor — DISMISSED** [review]

File: packages/components/organisms/f-cookie-banner/README.md [hunk 2]
```diff
@@ -77,13 +77,17 @@ Files can be accessed directly via CDN using [unkpg.com](https://unpkg.com/brows
...
-    <link
-      href="https://unpkg.com/@justeat/f-cookie-banner/dist/static/en-GB.css"
-      rel="stylesheet"
-    />
+    <!-- use fozzie CSS files to provide base styles and utilities classes used by the cookie banner -->
+    <link rel="stylesheet" href="https://unpkg.com/@justeat/fozzie/dist/css/fozzie-reset.css" />
+    <link rel="stylesheet" href="https://unpkg.com/@justeat/fozzie/dist/css/fozzie-typography.css" />
+    <link rel="stylesheet" href="https://unpkg.com/@justeat/fozzie/dist/css/fozzie-utilities.css" />
+    <!--  use the cookie banner CSS  -->
+    <link href="https://unpkg.com/@justeat/f-cookie-banner/dist/static/en-GB.css" rel="stylesheet" />
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-cookie-banner/README.md:90]: Does the tenant css change at all? Do we need to mention anything about usage in different tenants?

Comment (jamieomaguire) [reply packages/components/organisms/f-cookie-banner/README.md:90]: not to my knowledge!

Comment (jamieomaguire) [reply packages/components/organisms/f-cookie-banner/README.md:90]: added a comment in the example

**Pull request merged**